pkgs/by-name/ty/typeinc/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  nix-update-script,
+  installShellFiles,
+  versionCheckHook,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "typeinc";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "AnirudhG07";
+    repo = "Typeinc";
+    tag = "v${version}";
+    hash = "sha256-WxNI/2FNVYmeZC7YsaythYPjto+3bSCFOhP+OJ1tZw8=";
+  };
+
+  pyproject = true;
+  build-system = [ python3Packages.hatchling ];
+  nativeBuildInputs = [ installShellFiles ];
+
+  dependencies = with python3Packages; [
+    setuptools
+    pyttsx3
+    pyperclip
+    keyboard
+  ];
+
+  postInstall = ''
+    installManPage docs/man/typeinc.1
+  '';
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  doInstallCheck = true;
```

```suggestion
  checkInputs = [ versionCheckHook ];
```

Sorry. https://github.com/NixOS/nixpkgs/issues/420531

---

Did you see `Typeinc - version 1.0.1`? It comes from https://github.com/AnirudhG07/Typeinc/blob/main/pyproject.toml#L3. You can patch it in `postPatch`